.github/workflows/build.yml

```diff
@@ -31,11 +32,8 @@ jobs:
       - uses: ankane/setup-postgres@v1
         with:
           database: ${{ env.PGDATABASE }}
-      # Ensures that the database is in a clean state before any test runs.
       - name: "Show unique job ID"
         run: echo ${{ env.ACTIVE_STASH_TEST_COLLECTION_PREFIX }}
-      - name: "Clear out database"
```

Comment:
Since each job gets a fresh PG DB, we don't really need to clear it out.

Reply:
Would this cleanup be needed if this test failed and needed to be re run?

Reply:
I don't think so. I could be wrong, but it looks like each job gets a fresh, local PG DB.